### PR TITLE
Add GitHub PR and issue templates and other documentation files.

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
- Add GitHub PR and issue templates, feature doc template, release notes, contributing, code of conduct.
- Update Readme.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# How to contribute
+
+There are lots of ways to contribute to the Uno Platform and we appreciate the help from the community. You can provide feedback, report bugs, give suggestions, contribute code, and participate in the platform discussions.
```
Comment (jeromelaban) [inline CONTRIBUTING.md]: There are lots of ways to contribute to the _Source Generation Tasks_

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Code of conduct
+
+To better foster an open, innovative and inclusive community please refer to our [Code of Conduct](CODE_OF_CONDUCT.md) when contributing.
+
+## Provide feedback
```
Comment (jeromelaban) [inline CONTRIBUTING.md]: Remove this section

File: CONTRIBUTING.md (hunk 1 above)
Comment (agneszitte-nventive) [reply CONTRIBUTING.md]: I will update it thanks

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+**Identifying the scale**
+
+If you would like to contribute to one of our repositories, first identify the scale of what you would like to contribute. If it is small (grammar/spelling or a bug fix) feel free to start working on a fix.
+
+If you are submitting a feature or substantial code contribution, please discuss it with the team. You might also read these two blogs posts on contributing code: [Open Source Contribution Etiquette](http://tirania.org/blog/archive/2010/Dec-31.html) by Miguel de Icaza and [Don't "Push" Your Pull Requests](https://www.igvita.com/2011/12/19/dont-push-your-pull-requests/) by Ilya Grigorik. Note that all code submissions will be rigorously reviewed and tested by the Uno Platform team, and only those that meet an extremely high bar for both quality and design/roadmap appropriateness will be merged into the source.
+
+**Obtaining the source code**
+
+If you are an outside contributer, please fork the Uno Platform repository you would like to contribute to your account. See the GitHub documentation for [forking a repo](https://help.github.com/articles/fork-a-repo/) if you have any questions about this.
```
[the remark this answers is not in the document]
Comment (agneszitte-nventive) [reply CONTRIBUTING.md]: Will be updated, Thanks !

**Pull request merged**